fakeapp_test.go

```diff
@@ -67,7 +67,9 @@ func startFakeBookingApp() {
 	ERROR = TRACE
 
 	MainTemplateLoader = NewTemplateLoader([]string{ViewsPath, path.Join(RevelPath, "templates")})
-	MainTemplateLoader.Refresh()
+	if err := MainTemplateLoader.Refresh(); err != nil {
+		panic(err)
```

Comment:
t is not defined in startFakeBookingApp () and all the ERROR loggers are set to io.Discard